README.md

```diff
@@ -2,3 +2,106 @@
 
 Tikui is a MIT-licensed free software project allowing you to create a web pattern library.
 
+## Prerequisites
+
+* [Node.js](https://nodejs.org)
+
+## Development
+
+### Serve
+
+In development, you can run the application locally on [localhost:3000](http://localhost:3000/)
+
+```bash
+npm run serve
+```
+
+### Tutorial
+
+#### Create a component
+
+The source folder `src` follows the [Atomic Design](http://atomicdesign.bradfrost.com/table-of-contents/) methodology.
+
+To make our first component, we will take an example, an atom: a button.
+
+Inside `src/atom/atom.pug`, you have to describe your button by adding an inclusion:
+
+```pug
+include:componentDoc(height=55) button/button.md
+```
+
+Now, we have to create our atom by adding the button folder and the button documentation as a markdown file:
+
+```bash
+mkdir src/atom/button
+touch src/atom/button/button.md
+```
+
+In `button.md` file we can add:
+
+```markdown
+## Button
+
+A simple button.
+```
+
+Now, we can open [button](http://localhost:3000/atom.html#button) atom inside our browser (serve is needed).
+
+We can see a title `Button`, a content `A simple button` and two files to create:
+
+```bash
+touch src/atom/button/button.render.pug
+touch src/atom/button/button.template.pug
+```
+
+> The file `button.render.pug` represents the render of your component and `button.template.pug` represents his code.
+
+Inside `button.render.pug`, we can add:
+
+```pug
+extends /layout
+
+block body
+    include button.template.pug
+```
+
+And inside `button.template.pug`:
+
+```pug
+button.tikui-button Button
+```
+
+Then we can see a button on the browser. Now, we have to change the appearance of this button on `_atom.scss` and `_button.scss` files:
+
+```bash
+touch src/atom/_atom.scss
```

Comment:
What do you think of dividing this tutorial in two parts:
* The first one: Prepare the tikui for atomic design methodology.
* The second one: Create the atom button

It would separate the canvas creation which is done only once and the graphical contribution.

Reply:
Can we do it in a second iteration ?

Reply:
Yes why not.